CRAMv3.tex

```diff
@@ -1410,12 +1421,11 @@ \subsubsection*{Base substitution codes (BS data series)}
 
 AN: 3
 
-and they are written as a single byte, 10 01 00 11 = 147 decimal or 0x93 in this 
-case. The whole substitution matrix is written as 5 bytes, one for each reference 
-base in the alphabetical order: A, C, G, T and N.
+The substitution matrix entry for this reference base then is written as a single byte, with the
+codes in the order CGTN: 10 01 00 11 = 147 decimal, or 0x93 in this case.
 
-Note: the last two bits of each substitution code are redundant but still required 
-to simplify the reading. 
+Note that the last two bits of the substitution matrix entry are redundant, since the remaining
```

Comment:
Maybe we should just remove this sentence.  It really doesn't add anything and just raises questions. :)

If we keep it, then it should state that the last two bits of each substitution byte are redundant, rather than imply just the last 2 bits of the entire thing.

Reply:
Also done (removed this line).